Translation/DatabaseFreshResource.php

```diff
@@ -2,14 +2,14 @@
 
 namespace Lexik\Bundle\TranslationBundle\Translation;
 
-use Symfony\Component\Config\Resource\ResourceInterface;
+use Symfony\Component\Config\Resource\SelfCheckingResourceInterface;
```

Comment:
This interface seems to be added in `2.8` but the bundle should still be compatible with `2.7`.